src/Middleware/PageDecorationMiddleware.php

```diff
@@ -42,13 +42,15 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
             )->process($request, $handler);
         }
 
+        $DB = \Database::singleton();
```

Comment:
Could you change this to use the Factory declaration instead?

Reply:
done